docs/manage/security/rbac.rst

```diff
@@ -32,17 +32,18 @@ Brand new Determined installations include two user accounts:
    role.
 -  The ``determined`` user has no permissions.
 
-Both accounts have empty passwords. You are encouraged to set strong passwords or deactivate these
-accounts for security reasons.
+For security purposes, either create a strong password for both accounts or deactivate them.
+
```

Comment:
i think with the new security feature these will already have been set by the admin since this is no longer an additional step during setup